.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -1,38 +1,63 @@
 name: 🐛 Bug Report
-description: Create a report to help us improve this repo.
+description: Help us improve this repository by filing a detailed bug report.
 title: '[bug]: '
-labels: ['issue: bug']
+labels: [ 'issue: bug', 'status: to be confirmed' ]
 body:
   - type: markdown
     attributes:
       value: |
-        Thank you for filing a bug report!
-  - type: textarea
+        Bug Description |
+        A clear and concise description of the problem or unexpected behavior (attach screenshots/gifs if appropriate). |
+        Clicking on the "Save" button doesn’t save the form data. |
+  - type: markdown
     attributes:
-      label: What version of `@strapi/sdk-js` are you using?
-      placeholder: |
-        - Npm version
-        - Node.js version
-        - React version:
-        - `@strapi/sdk-js` version
-        - Browser
-    validations:
-      required: true
-  - type: textarea
+      value: |
+         Steps to Reproduce |
+         Provide a step-by-step description of how to reproduce the issue. Include code snippets or links to repositories when applicable.|
+         label: |
+           Example: |
+             1. Go to '...' |
+             2. Click on '...' |
+             3. Scroll down to '...' |
+             4. See error |
```

Comment:
The "go to"/"scroll to" might no be very relevant in our context, what do you think?

Reply:
it's just an example, I think it's fine. I can't think of a good example to use that would be code-based

Reply:
I guess it would be more like _"Use that model in your app, then generate your API schema, then make that call, then see the error in the console"_